mtags/src/main/scala/scala/meta/internal/docstrings/MarkdownGenerator.scala

```diff
@@ -104,9 +107,14 @@ object MarkdownGenerator {
       else "",
       if (c.see.nonEmpty)
         "\n**See**\n" + c.see
-          .map(
-            body => "- [" ++ blocksToMarkdown(body.blocks).trim + "]()"
-          )
+          .map { body =>
+            // If we already have a link, just render it
+            // otherwise create a link from the given text
+            blocksToMarkdown(body.blocks).trim match {
+              case LinkPattern(text) => s"- $text"
```

Comment:
Is it possible to move this regexp to the parser? I expected the link to be encoded in the `Comment` AST, it seems a bit unusual to do mini-parsing inside the printer

Reply:
Yes, makes sense. I'll try to move it into the parser.